webpack.config.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
   output: {
     path: Path.resolve(__dirname, "dist"),
     filename: "App.js",
-    chunkFilename: "[name].[contenthash].bundle.js",
```

Comment:
for future reference: we know this defeats cache busting for re-deploys; we'll trade off this for the simplicity of not implementing a reverse callback:
- hosting (user asks for html) -> functions (inserts metadata) -> hosting (to get vendor hash) -> response

and just keep it as:
- hosting (user asks for html) -> functions (inserts metadata) -> response